### PR TITLE
Fixes multiple tiny issues

### LINKED ISSUE #8740: Bugs Detected After Merging PR #8737 , PR #8706, PR #8680
Follow-up : https://github.com/ohcnetwork/care_fe/pull/8737 , https://github.com/ohcnetwork/care_fe/pull/8706 , https://github.com/ohcnetwork/care_fe/pull/8680

**Bugs**

- [x] typo's are missing in the patient details page

Image

- [x] when you try to use restore button in the log update page, it won't restore any data to the form

Image

- [x] Sign out button icon is misaligned

Image

- [x] Make sure the minimum height for the content area takes up the height of the viewport across the platform

Image

- [ ] The relative time for the "Restore" button on the form is calculated based on when the user entered and exited the form. For example, if the user accessed the registration form at 1:00 PM and reloaded the page at 1:05 PM, the relative time displayed on the button should indicate "a few seconds ago." However, it currently shows "5 minutes ago" because the relative time is being calculated from the moment the user first entered the form, rather than from when the page was reloaded.

Image

- [x] The breadcrumb navigation isn't functioning as expected, preventing users from returning to the immediate previous page. For instance, when a user is on the consent record page (which is part of the consultation page), clicking on the "admitted on..." breadcrumb should ideally take them back to the patient consultation page, but the now its not even clickable. This similar issue is occurring across the platform, where navigating back to the most recent previous page via breadcrumbs is consistently broken

Image

- [x] The Favicon is stretched. Use a 1:1 Favicon ratio

**Enhancements**

- [x] the width of the autocomplete text box and add button in Slideover such as linked facility, and linked skills, are mispositioned, increasing the text field width should fix it
Image

- [x] the card in the location management page and the page heading is misaligned, this similar issues of padding and heading getting misalinged is observed in other pages as well such as bed management
Image

- [x] Only show one loader for the entire page, rather than individual loader for each card in the facility location page 
Image

- [x] Remove the green bg for the collapse chevron in the sidebar. Just keep the chevron as black, and add a light gray bg on hover.

Image

- [x] add border colour to the border of covid details section, since the separation is not properly visible 

Image

### LINKED ISSUE #8752: Discharged page jumps to page 1
**Describe the bug**
Whenever any patient details is viewed in page other than 1, on returning back, the dashboard returns to page 1, which is not an expected behaviour ; It does not happens with live patients.

**To Reproduce**

1. Go to 'patients tab'
2. Click on 'discharged' tab and click on any facility
3. Scroll down to 'page index'
4. move to any other page than 1
5. click the patient to view details
6. move back
7. see error

**Expected behavior**
The page must not jump to page 1.

**Recording**
https://github.com/user-attachments/assets/0d022933-a57f-4171-b1a2-39cc1d2173c6

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Fixes #8740
- Fixes #8752

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
image

- [x] the text badge overlaps with patient card details in mobile view when there is no bed assigned to the patient

image
image

- [x] in the mobile view, when there is no facility cover image, its still showing the old icon

image

- [x] the collapse icon is not working in the mobile view

**Comment by nihal467** [comment]
@rithviknishad can you fix the minor bugs mentioned above, rest of them looks good

**Comment by nihal467** [comment]
@rithviknishad 
image

- [x] in the mobile view, keep the card size content in the patient list page

File: src/Routers/AppRouter.tsx [hunk 1]
```diff
@@ -146,7 +146,7 @@ export default function AppRouter() {
             id="pages"
             className="flex-1 overflow-y-scroll bg-gray-100 pb-4 focus:outline-none md:py-0"
           >
-            <div className="max-w-8xl mx-auto mt-4 rounded-t-lg border bg-gray-50 p-3 shadow-lg">
+            <div className="max-w-8xl mx-auto mt-4 min-h-[96vh] rounded-lg border bg-gray-50 p-3 shadow">
```
Comment (bodhish) [inline src/Routers/AppRouter.tsx:149]: Its fixed in #[8789](https://github.com/ohcnetwork/care_fe/pull/8789)

File: src/Locale/en.json [hunk 2]
```diff
@@ -1022,9 +1022,12 @@
   "date_of_result": "Covid confirmation date",
```
Comment (bodhish) [inline src/Locale/en.json:1022]: We should have a pre-commit hook that sorts en.yml, (another issue for sure)

**Review by bodhish — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**